powertools-sqs/src/main/java/software/amazon/lambda/powertools/sqs/SqsBatch.java

```diff
@@ -32,6 +32,19 @@
  * {@link SqsBatch#suppressException()} to true. By default its value is false
  * </p>
  *
+ * <p>
+ * If you want certain exceptions to be treated as permanent failures, i.e. exceptions which are not worth retrying and
```

Comment:
I think we should change the phrase "not worth retying" to something like "exceptions where the result of retrying will always be a failure"